content/en/serverless/google_cloud_run/_index.md

```diff
@@ -107,96 +140,154 @@ CMD ["rails", "server", "-b", "0.0.0.0"] (adapt this line to your needs)
 See [Tracing Ruby Applications][1] for detailed instructions. [Sample code for a simple Ruby application][2].
 
 [1]: /tracing/trace_collection/dd_libraries/ruby/
+
 [2]: https://github.com/DataDog/crpb/tree/main/ruby-on-rails
 {{< /programming-lang >}}
 {{< /programming-lang-wrapper >}}
 
-#### Troubleshooting
-This integration depends on your runtime having a full SSL implementation. If you are using a slim image for Node, you may need to add the following command to your Dockerfile to include certificates.
+Then build the container with
+
+```shell
+docker build --tag gcr.io/YOUR_PROJECT/YOUR_APP_NAME .
 
-```
-RUN apt-get update && apt-get install -y ca-certificates
 ```
 
-#### Build with the Datadog buildpack
+#### Using Datadog buildpack
 
-1. Use [`pack`][4] to build your application by running the following:
-   ```
+[`Pack Buildpacks`][4] provide a convenient way to package your container without using a Dockerfile. This example
+will use the GCP container registry and Datadog serverless buildpack. Build your application by running the following
+command making sure to use the latest tag for [`datadog/serverless-buildpack`][13]:
+
+   ```shell
    pack build --builder=gcr.io/buildpacks/builder \
    --buildpack from=builder \
    --buildpack datadog/serverless-buildpack:beta4 \
    gcr.io/YOUR_PROJECT/YOUR_APP_NAME
-
    ```
 
-   **Note**: Not compatible with Alpine.
+**Note**: Not compatible with Alpine.
 
- 2. Push your image to GCP:
-    ```
-    docker push gcr.io/YOUR_PROJECT/YOUR_APP_NAME
-    ```
+### Push the container image to the registry
+
+Make sure you are logged in `gcloud` and authorized docker with `gcloud auth configure-docker`. Then push to your GCP
+container registry:
+
+   ```shell
+   docker push gcr.io/YOUR_PROJECT/YOUR_APP_NAME
+   ```
 
 ### Deploy to Cloud Run
```

Comment:
Also we should add instructions for DD_SITE as well, like this https://github.com/DataDog/documentation/blob/master/content/en/serverless/installation/python.md?plain=1#L314. It is also mandatory, otherwise people run into invalid api key error. We have seen many people run into this problem in the past.

Reply:
good point. Added `DD_TRACE_ENABLED` too